src/__tests__/CompareResults/ResultsView.test.tsx

```diff
@@ -16,6 +16,8 @@ import {
   FetchMockSandbox,
 } from '../utils/test-utils';
 
+jest.setTimeout(60000);
```

Comment:
you probably don't need this change :-) right? Can you explain why you added these?

Reply:
I added the jest.setTimeout(60000) to prevent timeout issues during tests as they were failing because of it. The only way I could solve it and have all tests passed